server/src/internalClusterTest/java/org/opensearch/action/admin/indices/datastream/DataStreamTestCase.java

```diff
@@ -67,6 +77,12 @@ public RolloverResponse rolloverDataStream(String name) throws Exception {
         RolloverResponse response = client().admin().indices().rolloverIndex(request).get();
         assertThat(response.isAcknowledged(), is(true));
         assertThat(response.isRolledOver(), is(true));
+        if (performRemoteStateRestore()) {
+            String clusterUUIDBefore = clusterService().state().metadata().clusterUUID();
+            resetCluster();
+            String clusterUUIDAfter = clusterService().state().metadata().clusterUUID();
+            assertFalse(clusterUUIDBefore.equals(clusterUUIDAfter));
+        }
```

Comment:
we can move this as protected method in OpenSearchIntegTestCase class so that we can we can use it for others tests as well

Reply:
Moved it to `OpenSearchIntegTestCase`.